Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -332,5 +450,4 @@ void vtkThreadedImageAlgorithm::ThreadedExecute(
   (void)extent;
   (void)threadId;
   vtkErrorMacro("Subclass should override this method!!!");
-}
-
+}
```

Comment:
This little symbol in github means that the last line in the file doesn't end with a "newline" character (the C++ standard has a rule that the last character in every source file ends with a newline, but unfortunately Visual Studio ignores this rule).